src/ui/sass/modules/_SearchBar.scss

```diff
@@ -121,6 +121,7 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
   &-clear
   {
     display: flex;
+    flex-shrink: 0;
```

Comment:
Since we removed the `flex-grow` from `&-input` is this still necessary?

Reply:
I tested this PR without flex-shrink it and the buttons still shrink at low resolutions. The buttons and the search bar items are all flex items because their parent (the search bar form) has `display: flex;` set. By default the flex items have a flex-shrink of 1, which makes them susceptible to shrinking. Setting flex-shrink to 0 prevents that.